dev/com.ibm.ws.jaxrs.2.1_fat_extended/fat/src/com/ibm/ws/jaxrs21/fat/extended/PackageJsonBTestNoFeature.java

```diff
@@ -23,12 +23,13 @@
 import componenttest.annotation.SkipForRepeat;
 import componenttest.annotation.TestServlet;
 import componenttest.custom.junit.runner.FATRunner;
+import componenttest.rules.repeater.JakartaEE10Action;
 import componenttest.rules.repeater.JakartaEE9Action;
 import componenttest.topology.impl.LibertyServer;
 import componenttest.topology.utils.FATServletClient;
 import jaxrs21.fat.jsonb.JsonBTestServlet;
 
-@SkipForRepeat(JakartaEE9Action.ID) // failures when comparing JSON
+@SkipForRepeat({JakartaEE9Action.ID, JakartaEE10Action.ID}) // failures when comparing JSON
```

Comment:
should probably have an issue for this one too... but probably lower priority

Reply:
https://github.com/OpenLiberty/open-liberty/issues/22520